packages/administration/composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "shopsys/administration",
+    "type": "library",
+    "description": "Shopsys Platform administration",
+    "keywords": ["administration", "Shopsys Platform", "SSFW", "SSP"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Shopsys",
+            "homepage": "https://www.shopsys.com/"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Shopsys\\Administration\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\Administration\\": "tests/"
+        }
+    },
+    "require": {
+        "php": "^8.3",
+        "ext-json": "*",
+        "cweagans/composer-patches": "^1.7",
+        "sonata-project/admin-bundle": "^4.30",
+        "sonata-project/doctrine-orm-admin-bundle": "^4.9",
+        "twig/twig": "^3.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5.20",
+        "shopsys/coding-standards": "15.0.x-dev"
+    },
+    "config": {
+        "allow-plugins": {
+            "symfony/flex": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "cweagans/composer-patches": true
+        }
+    },
+    "extra": {
+        "enable-patching": true,
+        "patches": {
+            "composer-exit-on-patch-failure": true,
+            "twig/twig": {
+                "Hotfix for issue https://github.com/sonata-project/SonataAdminBundle/issues/8181": "packages/administration/src/Twig/sonata.patch"
```

Comment:
the path in the context of the administration bundle will be different – `"src/Twig/sonata.patch"`

Reply:
I have moved it to project-base